athena_cli.py

```diff
@@ -77,7 +93,7 @@ def execute(self, statement):
             time.sleep(0.2)  # 200ms
 
         if status == 'SUCCEEDED':
-            output_results(self.athena, self.format, execution_id, sys.stdout)
+            output_results(self.athena, self.format, execution_id, sys.stdout, False)
```

Comment:
For boolean parameters I would include the paramater name eg `..., is_shell=False)`.

Reply:
Yes, agreed. Done.